src/audioPlayer.js

```diff
@@ -100,14 +109,49 @@ async function setPlayer() {
 }
 
 function resetControllers() {
-    AUDIO_TOGGLES.forEach((audioToggle) => {
-        const playIconPlayer = audioToggle.querySelector(
-            '[data-element=play]'
-        )
-        const pauseIconPlayer = audioToggle.querySelector(
-            '[data-element=pause]'
-        )
-        playIconPlayer.setAttribute('display', 'block')
-        pauseIconPlayer.setAttribute('display', 'none')
+    [...AUDIO_TOGGLES, ...VIDEO_TOGGLES].forEach((mediaToggle) => {
+        const playIcon = mediaToggle.querySelector('[data-element=play]')
+        const pauseIcon = mediaToggle.querySelector('[data-element=pause]')
+        const unmuteIcon = mediaToggle.querySelector('[data-element=unmute]')
+        const muteIcon = mediaToggle.querySelector('[data-element=mute]')
+
+        if (playIcon) playIcon.setAttribute('display', 'block')
+        if (pauseIcon) pauseIcon.setAttribute('display', 'none')
+        if (unmuteIcon) unmuteIcon.setAttribute('display', 'block')
+        if (muteIcon) muteIcon.setAttribute('display', 'none')
+    })
+}
+
+
+/**
+ * A helper function to toggle a video's mute state. Bind the trigger to this function so that the video selector can be
+ * safely retrieved and the icon in the button can be changed.
+ */
+function toggleVideoMute() {
+    const videoToggle = this
+
+    // Ensure the audio player and video audio do not play on top of each other.
+    PLAYER.pause()
+    resetControllers()
+
+    // If we add a case where the trigger's parent element does not also hold the video, this selector will need to be
+    // revised.
+    const video = $(videoToggle).siblings().find('video')
```

Comment:
What happens if Jquery cannot find the video? Does it break the page or will it only create console errors?

Reply:
Good question, I will test to make sure.
Either way, probably worth adding a Sentry alert if the video isn't found!